src/node.rs

```diff
@@ -244,12 +244,20 @@ impl<S: StorageExt> SawtoothRaftNode<S> {
 
         let is_leader = self.raw_node.raft.state == raft::StateRole::Leader;
         if is_leader {
-            // We just became the leader, so we need to start building a block
+            // We just became the leader
             if self.leader_state.is_none() {
-                debug!("Leader({:?}) became leader, intializing block", self.peer_id);
+                // Check if we stepped up as leader from follower state
+                if let Some(FollowerState::Committing(ref block_id)) = self.follower_state {
+                    debug!("Follower({:?}) became leader while it was in committing state",
+                        self.peer_id);
+                    self.leader_state = Some(LeaderState::Committing(block_id.clone()));
+                } else  {
```

Comment:
There are two spaces after the `else`

Reply:
Will Update. Does lint (rust fmt) complain on this ?

Reply:
I believe rustfmt does, yes

Reply:
Updated commit